netket/optimizer/qgt/qgt_onthefly.py

```diff
@@ -29,7 +29,7 @@
 from ..linear_operator import LinearOperator, Uninitialized
 
 
-def QGTOnTheFly(vstate=None, **kwargs) -> "QGTOnTheFlyT":
+def QGTOnTheFly(vstate=None, *, chunk_size=None, **kwargs) -> "QGTOnTheFlyT":
```

Comment:
I think the added function argument should be documented below in the `Args` section. At least with a reference to `vstate.chunk_size` and ideally with a brief note under which circumstances one would want to use a different chunk size for the QGT.

(This comment applies to the other two QGT constructor methods too.)